fetch_maps/package.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<package>
+  <name>fetch_maps</name>
+  <version>0.0.1</version>
+  <description>The fetch_maps package</description>
+
+  <maintainer email="ablasdel@fetchrobotics.com">Aaron Blasdel</maintainer>
```

Comment:
@ablasdel you can make yourself an author if you want, but you should probably put me as the maintainer so that I get release failures (or at least add me as a maintainer)